### PR TITLE
ADEN-4137 | track response time for prebid bidders in lookup events 

### PR DESCRIPTION
**Pull request by bognix**

### CODE REVIEW

File: extensions/wikia/AdEngine/js/AdTracker.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
 /*jshint camelcase:false*/
-define('ext.wikia.adEngine.adTracker', ['wikia.tracker', 'wikia.window', 'wikia.log'], function (tracker, window, log) {
+define('ext.wikia.adEngine.adTracker', ['ext.wikia.adEngine.utils.timeBuckets','wikia.tracker', 'wikia.window', 'wikia.log'], function (timeBuckets, tracker, window, log) {
 	'use strict';
 
-	var timeBuckets = [0.0, 0.5, 1.0, 1.5, 2.0, 2.5, 3.5, 5.0, 8.0, 20.0, 60.0],
+	var buckets = [0.0, 0.5, 1.0, 1.5, 2.0, 2.5, 3.5, 5.0, 8.0, 20.0, 60.0],
 		logGroup = 'ext.wikia.adEngine.adTracker';
 
 	function encodeAsQueryString(extraParams) {
```
Comment (WikiaTech) [inline extensions/wikia/AdEngine/js/AdTracker.js:9]: MAJOR Function has a complexity of 12 which is greater than 10 authorized. [rule](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AFunctionComplexity)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue
- MAJOR 1 major

Watch the comments in this conversation to review them.

**Review by v3rth — APPROVED** [review]
LGTM

**Comment by fraszczakszymon** [comment]
LGTM 👍

**Pull request merged**